### PR TITLE
feat(infra): Add workflow for IDS

### PR DESCRIPTION
**Pull request by peturgq**
# ...

Attach a link to issue if relevant

## What

Deploy supporting IDS services in workflow

## Why

Because these services are now ubercharted with the IDS we need to deploy them with IDS when these services are changed.
 
## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -674,7 +673,28 @@ jobs:
             }
           }
           BODY
-
+      - uses: actions/checkout@v3
```
Comment (pshomov) [inline .github/workflows/push.yml:676]: Not needed action.

**Review by pshomov — CHANGES_REQUESTED** [review]
Some minor improvements needed

File: .github/workflows/push.yml [hunk 2]
```diff
@@ -654,7 +654,6 @@ jobs:
       matrix:
         service:
           - islandis
-          - identity-auth-api
```
[the remark this answers is not in the document]
Comment (peturgq) [reply .github/workflows/push.yml:657]: Fixed in [662da9f](https://github.com/island-is/island.is/pull/7703/commits/662da9fe325ce0d2b32ee1c69694d1ea86921339)

File: .github/workflows/push.yml (hunk 1 above)
Comment (peturgq) [reply .github/workflows/push.yml]: Fixed in [662da9f](https://github.com/island-is/island.is/pull/7703/commits/662da9fe325ce0d2b32ee1c69694d1ea86921339)

**Review by pshomov — APPROVED** [review]
LGTM

**Pull request merged**